package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
do we do this in main or in develop ?

Reply:
true. usually only in develop if a migration is needed. will revert